npm/webpack-dev-server/src/makeWebpackConfig.ts

```diff
@@ -72,6 +77,67 @@ function modifyWebpackConfigForCypress (webpackConfig: Partial<Configuration>) {
   return webpackConfig
 }
 
+const versionRe = /(\d.+?)\./
```

Comment:
Same comment regarding `semver` here - I'd like it if we could use the lib just to avoid the logic duplication, but I get that adding a Node dependency isn't always desirable

Reply:
I like to keep the `cypress/{vue,react}` bundles really lean, but we definitely need a lot more tests around this regexp. I might grab the relevant code from the semver library, starting w/ the regexp  and test cases perhaps: https://gist.github.com/jhorsman/62eeea161a13b80e39f5249281e17c39

Reply:
Let's just use semver! c8fee62